install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: metal3provisionings.metal3.io
```

Comment:
I don't have strong opinions on how you want to shape this api and, mao just happens to be the integration point for this today so I'm good merging this as soon as test goes green but as an observation `metal3provisionings` seems odd t me,`metal3` is already in the domain so just `provisionings.metal3.io` would be more meaningful imho

Reply:
Having a plural "provisionings" also seems strange. should it be just provisioning.metal3.io?

Reply:
@enxebre I was stuck here. Since this is a singleton CR, plural really does not make sense but the unit tests fail without a value for plural.
Without Metal3, the CRD would be called Provisioning. I am pushing another version with these changes.